Update signing-android.md (#493)
codemagic-ci-cd/codemagic-docs: content/yaml-code-signing/signing-android.md

  }
  ...
```

> 2. Add your keystore and keystore details in the `Environment variables` section of the Codemagic UI. [`Secure`](../building/encrypting/#encrypting-sensitive-data) your keystore file, keystore password (if keystore is password-protected), key alias and key alias password (if key alias is password-protected) and set the values to the following environment variables. Note that securing files requires you to [`base64 encode`](https://docs.codemagic.io/variables/environment-variable-groups/#global-variables-and-secrets) them locally and decode them during the build.

Reviewer: We normally use bold to highlight UI elements, so it should be **Environment variables** and **Secure**. Also note that the link for Secure points to a file we no longer have, the right link would be this: ../variables/environment-variable-groups/.
Also we should use relative links everywhere in docs and not full links. 
The base64 encode link currently do not point to the right section either, it should be this: ../variables/environment-variable-groups/#storing-sensitive-valuesfiles. So actually it looks like both links should take to the same section.

Reviewer: I think the whole section should be expanded a bit. The variable names are still important, right? The way I would present the instructions is sth like this:
- Add your environment variables in Codemagic UI (either in application or team variables)
- The names of the variables and what to add there
- A note about encoding binary files and decoding them in yaml
- A note about checking **Secure**
- A note about saving them all to the same group so they can be important to yaml at once

  }
  ...
```

> 2. Save the keystore file, keystore password (if keystore is password-protected), key alias and key alias password (if key alias is password-protected) to the respective environment variables in the **Environment variables** section in Codemagic UI. Click **Secure** to encrypt the values. Note that binary files i.e. keystore file requires you to [`base64 encode`](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) them locally, before they can be saved to environment variables and decode them during the build.

Reviewer: ```suggestion
2. Save the keystore file, keystore password (if keystore is password-protected), key alias and key alias password (if key alias is password-protected) to the respective environment variables in the **Environment variables** section in Codemagic UI. Click **Secure** to encrypt the values. Note that binary files (i.e. keystore) have to be [`base64 encoded`](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) locally before they can be saved to environment variables and decoded during the build.
```

```
### Set up code signing with user-specified keys

In order to do code signing [`Secure`](../building/encrypting/#encrypting-sensitive-data) your keystore file, keystore password (if keystore is password-protected), key alias and key alias password (if key alias is password-protected) and set the values to the following environment variables.:

```
FCI_KEYSTORE_PATH: /tmp/keystore.keystore
FCI_KEYSTORE
FCI_KEYSTORE_PASSWORD
FCI_KEY_ALIAS

> FCI_KEY_PASSWORD

Reviewer: These variables need to be added in the UI as well and not in yaml

```
### Set up code signing with user-specified keys

> In order to do code signing Save the keystore file, keystore password (if keystore is password-protected), key alias and key alias password (if key alias is password-protected) to the respective environment variables in the **Environment variables** section in Codemagic UI. Click **Secure** to encrypt the values. Note that binary files i.e. keystore file requires you to [`base64 encode`](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) them locally, before they can be saved to environment variables and decode them during the build:

Reviewer: ```suggestion
In order to code sign the build, save the keystore file, keystore password (if keystore is password-protected), key alias and key alias password (if key alias is password-protected) to the respective environment variables in the **Environment variables** section in Codemagic UI. Click **Secure** to encrypt the values. Note that binary files (i.e. keystore) have to be [`base64 encode`](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) locally before they can be saved to environment variables and decoded during the build:
```
